science/xraylib/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup           github 1.0
 PortGroup           compilers 1.0
 
 github.setup        tschoonj xraylib 4.1.2 xraylib-
-revision            0
+revision            1
```

Comment:
adding newer perl versions (just as is the case with Python subports) does not require an increase in `revision`